mobile/lib/pages/editing/crop.page.dart

```diff
@@ -0,0 +1,160 @@
+          return Column(
+            children: [
+              Container(
+                padding: const EdgeInsets.only(top: 20),
+                width: constraints.maxWidth * 0.8,
+                height: constraints.maxHeight * 0.6,
+                child: CropImage(
+                  controller: _cropController,
+                  image: widget.image,
+                  gridColor: Colors.white,
+                ),
+              ),
+              Expanded(
+                child: Container(
+                  width: double.infinity,
+                  decoration: const BoxDecoration(
+                    color: Colors.black,
+                    borderRadius: BorderRadius.only(
+                      topLeft: Radius.circular(20),
+                      topRight: Radius.circular(20),
+                    ),
+                  ),
+                  child: Center(
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.only(
+                              left: 20, right: 20, bottom: 10),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                            children: [
+                              IconButton(
+                                icon: const Icon(Icons.rotate_left,
+                                    color: Colors.white),
+                                onPressed: () {
+                                  _cropController.rotateLeft();
+                                },
+                              ),
+                              IconButton(
+                                icon: const Icon(Icons.rotate_right,
+                                    color: Colors.white),
+                                onPressed: () {
+                                  _cropController.rotateRight();
+                                },
+                              ),
+                            ],
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: <Widget>[
+                            _buildAspectRatioButton(context, null, 'Free'),
+                            _buildAspectRatioButton(context, 1.0, '1:1'),
+                            _buildAspectRatioButton(
+                                context, 16.0 / 9.0, '16:9'),
+                            _buildAspectRatioButton(context, 3.0 / 2.0, '3:2'),
+                            _buildAspectRatioButton(context, 7.0 / 5.0, '7:5'),
+                          ],
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ],
+          );
+        },
+      ),
+    );
+  }
+
+  Widget _buildAspectRatioButton(
```

Comment:
Make this its own widget.

```dart
class AspectRatioButton extends StatelessWidget
```

You could also make it privately scoped, if you'd prefer:

```dart
class _AspectRatioButton extends StatelessWidget
```

if you only want it to live in this class and nowhere else.